x-pack/plugin/eql/qa/correctness/src/javaRestTest/java/org/elasticsearch/xpack/eql/EsEQLCorrectnessIT.java

```diff
@@ -105,18 +77,6 @@ public static void logTotalExecutionTime() {
         LOGGER.info("Total time: {} ms", totalTime);
     }
 
-    @AfterClass
```

Comment:
Why removing this method?

Reply:
Because when you have a running ES node and you try to execute a single query against it with gradle, the termination of the suite will delete the data, and you'd have to rerun the EqlDataLoader.